src/main/api/2/studio-api-2.yaml

```diff
@@ -2988,6 +2994,110 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/children:
+    get:
+      tags:
+        - content
+      summary: Get list of children for given content path
+      description: 'Required permission "get_children" ("site member")'
+      operationId: getChildren
+      parameters:
+        - name: siteId
+          in: query
+          description: Site ID
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: content path to get children for
+          required: true
+          schema:
+            type: string
+        - name: locale
+          in: query
+          description: filter children by locale, or show source items (not translation) if translatation to specified locale does not exist
+          schema:
+            type: string
+        - name: sortStrategy
+          in: query
+          description: sort strategy (sort order)
+          schema:
+            type: string
+        - name: offset
+          in: query
+          description: offset of first child in response - default 0
+          schema:
+            type: integer
+        - name: limit
+          in: query
+          description: number of children to return
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/SidebarItem'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
```

Comment:
Please add 403 since we want to support that down the road. Another option is to punt now and do it later. Let me know what you think.